content/en/admin/tootctl.md

```diff
@@ -658,6 +667,7 @@ Remove locally cached copies of media attachments from other servers.
 **Version history:**\
 2.5.0 - added\
 2.6.2 - show freed disk space
+4.1.0 - added --prune_profiles, --remove-headers, and --include-follows.
```

Comment:
I believe dash is preferred to underscore.

Reply:
Same as the other, I'm confused on what the actual command is. I'm not a ruby expert. Your PR shows the options as underscore but to use them you have to use dash?